apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -4062,6 +4062,10 @@
       "title": "Sorry, internet seems to be down",
       "description": "Please check your internet connection."
     },
+    "NetworkError": {
```

Comment:
Isn't there any similar error message type like this ?

Reply:
Yes there is the NetworkDown error but we discussed with Hakim that we may need another error that is related to unique calls failing and not the whole network

Reply:
Work in progress because we discovered the backend sends errors even if it's because balance is zero or there is no other account retrievable

Need to do like further down in the code a switch / series of if in the catch clause